### ISSUE TITLE
Shadowing users

### ISSUE THREAD

**Issue by aled2305** [sentora/sentora-core#107]
Hi Bobby :) As requested I've added the bug here...

Hi, I have just found a bug with the shadowing feature..

If I navigate in the following sequence;

Administrator>>Reseller>>Client>>

 I will take me to the client as expected, but when I click end shadowing it will take me back to the Reseller... and it also thinks I've signed in as reseller. This means I must sign out of reseller and sign back in as admin.

Basically what would be ideal is when end shadowing is clicked it will go back to reseller where I can click end shadowing again to go back to admin, or from client just go strait back to admin. This is assuming I am Admin and not reseller :)

**Comment by MBlagui** [comment]
Unable to reproduce it.

**Comment by TGates71** [comment]
Confirmed.
Admin>reseller>user works.
user>reseller>admin broken (Stops at reseller)
user>reseller works

**Comment by bobsta63** [comment]
Cool, thanks for confirming the issue, will fix and get released as part of 1.1.0 release.

**Comment by TGates71** [comment]
Does this problem still persist? Can somebody check it out?

**Comment by Dukecitysolutions** [comment]
Any news @TGates71 ?

**Comment by TGates71** [comment]
My recent observation and notes:
Suggestion: the shadowing bar, we should add the account name that you are shadowing.

If you shadow accounts, you can see your reseller client's accounts. (basically all accounts on the server)
Although you can't see them in the manage clients. which is OK since they are not your clients and zadmin should be able to check on all client accounts.

I think we need to add a notice in the shadow list that shows who the reseller is or make it so shadowing only shows your clients as zadmin, then you can shadow the reseller client and shadow their clients.

**Comment by TGates71** [comment]
Added the shadowed account user name to the bar. Will send you the code.
